google/cloud/apigee/registry/applications/v1alpha1/registry_styleguide.proto

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.cloud.apigee.registry.applications.v1alpha1;
+
+import "google/api/field_behavior.proto";
+
+option java_package = "com.google.cloud.apigee.registry.applications.v1alpha1";
+option java_multiple_files = true;
+option java_outer_classname = "RegistryStyleGuideProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// StyleGuide defines a set of guidelines, rules, and linters that govern
+// the static analysis of API Specs.
+message StyleGuide {
+    // Resource name of the style guide.
+    string name = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // A list containing style (format) descriptors for this style guide that 
+    // are specified as a Media Type (https://en.wikipedia.org/wiki/Media_type).
+    // Possible values include "application/vnd.apigee.proto", 
+    // "application/vnd.apigee.openapi", and "application/vnd.apigee.graphql",
+    // with possible suffixes representing compression types. These hypothetical
+    // names are defined in the vendor tree defined in RFC6838 
+    // (https://tools.ietf.org/html/rfc6838) and are not final.
+    repeated string mime_types = 2 [
+		(google.api.field_behavior) = REQUIRED
+	];
+
+    // A list of guidelines that are specified by this style guide.
+    repeated Guideline guidelines = 3;
+
+    // A list of linters that this style guide uses.
+    repeated Linter linters = 4;
+}
+
+message Guideline {
+    // Resource name of the guideline.
+    string name = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // Human-meaningful name.
+    string display_name = 2 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // A detailed description of the guideline.
+    string description = 3;
+
+    // A list of rules that this guideline specifies.
+    repeated Rule rules = 4;
+
+    // Indicates the status of the guideline.
```

Comment:
Nit: This seems like it should be moved to document the `status` field instead of the `Status` enum definition. The enum might be better described as the possible statuses of a guideline.

This feedback applies similarly to the `Rule` severity type and field definitions.